interface/patient_file/encounter/trend_form.php

```diff
@@ -98,8 +98,11 @@ function show_graph(table_graph, name_graph, title_graph)
         },
         error: function() {
             // hide the chart div
-          $('#chart').hide();  
-          alert(title_graph + " " + "<?php echo xlt('has no data to graph');?>" + ".\n" +"<?php echo xlt('Please select an item that has data');?>" + ".");
+          $('#chart').hide();
+          if(!title_graph){
+              title_graph = 'This item';
```

Comment:
To make this translatable, best to place the entire alert here:
``````
alert("<?php echo xlt('This item does not have enough data to graph');?>" + ".\n" +"<?php echo xlt('Please select an item that has more data');?>" + ".");
``````
And then place the below alert() behind and `else` clause

Reply:
You're right, my bad.